tests/src/test/java/org/wso2/micro/gateway/tests/common/ResponseConstants.java

```diff
@@ -25,16 +25,15 @@
     public static final String responseBodyV1 = "{\"id\":222222222, \"category\":{ \"id\":1, \"name\":\"Test\" }," +
             " \"name\":\"doggie\", \"photoUrls\":[ \"SampleImage1.png\" ], \"tags\":[ { \"id\":0, " +
             "\"name\":\"TestTag1\" } ], \"status\":\"pending\"}";
-    public static final String petByIdResponse = "{\"id\": 9199424981609281000," + "  \"category\": {"
-            + "    \"id\": 0," + "    \"name\": \"string\"" + "  }," + "  \"name\": \"doggie\","
-            + "  \"photoUrls\": [" + "    \"string\"" + "  ]," + "  \"tags\": [" + "    {"
-            + "      \"id\": 0," + "      \"name\": \"string\"" + "    }" + "  ],"
-            + "  \"status\": \"available\"" + "}";
-    public static final String petByIdResponseV1 = "{\"id\": 33333333333333," + "  \"category\": {"
-            + "    \"id\": 0," + "    \"name\": \"string\"" + "  }," + "  \"name\": \"doggieNew\","
-            + "  \"photoUrls\": [" + "    \"string\"" + "  ]," + "  \"tags\": [" + "    {"
-            + "      \"id\": 0," + "      \"name\": \"string\"" + "    }" + "  ],"
-            + "  \"status\": \"available\"" + "}";
+    public static final String petByIdResponse = "{\"id\":2, \"category\":{\"id\":1, \"name\":\"John Doe\"}, " +
+            "\"name\":\"shre\", \"photoUrls\":[\"ArrayItem1\"], \"tags\":[{\"id\":1, \"name\":\"TfNSW\"}]," +
+            " \"status\":\"hello\"}";
+    public static final String petByIdResponseV1 = "{\"id\":2, \"category\":{\"id\":1, \"name\":\"John Doe\"}, " +
+            "\"name\":\"shre\", \"photoUrls\":[\"ArrayItem1\"], \"tags\":[{\"id\":1, \"name\":\"TfNSW\"}], " +
+            "\"status\":\"hello\"}";
+    public static final String getPetResponse = "{\"id\":2, \"category\":{\"id\":1, \"name\":\"John Doe\"}," +
+            " \"name\":\"shre\", \"photoUrls\":[\"ArrayItem1\"], \"tags\":[{\"id\":1, \"name\":\"TfNSW\"}]," +
+            " \"status\":\"hello\"}";
```

Comment:
Isn't these responses used by other test cases to validate

Reply:
I checked other usages and verified it doesn't affect the functionality.